target/snitch_cluster/sw/apps/snax-mac/data/data.h

```diff
@@ -1,7 +1,7 @@
 uint32_t VEC_LEN = 20;
-uint32_t A[] = {99, 67, 39, 26, 62, 14, 17, 18, 54, 16,
+uint64_t A[] = {99, 67, 39, 26, 62, 14, 17, 18, 54, 16,
                 44, 9,  26, 85, 72, 66, 95, 65, 43, 84};
-uint32_t B[] = {86, 10, 14, 11, 38, 41, 94, 82, 97, 25,
+uint64_t B[] = {86, 10, 14, 11, 38, 41, 94, 82, 97, 25,
                 96, 71, 44, 59, 93, 38, 57, 21, 84, 29};
```

Comment:
Can we add an extra accelerator invocation later with signed/negative numbers?

Reply:
Good point, I don't think I've ever tried signed values in this MAC engine. I will try 🛩️

Reply:
@JosseVanDelm I will add this in a separate PR or test. I need time to investigate this part 🕵️